GoogleForCreators/web-stories-wp
# Fix: Media Accessibility Panel UI Simplifications

See: #9465

## Context

Media Accessibility Panel UI Simplifications

## Summary

Added description above the textarea for both Video and Image (instead of the current placeholder). 
See: [figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor---Sprint-Feature-Level-Designs_?node-id=10250%3A436662)


## Relevant Technical Choices



## To-do

 - [x] Accessibility panel: Add description above the textarea for both Video and Image (instead of the current placeholder).
 - [x] Filter Panel: Add eyedropper next to the Color input (not seen in Figma but it was confirmed by UX that it should be like this everywhere. Makes sense to add it to the Color input directly if not done elsewhere.

## User-facing changes

Description moved to the label from placeholder.

Before | After
---|---
<img src="https://i.imgur.com/h6KPx3M.png" alt="alt text" width="300" height="auto" />|<br/> <img src="https://i.imgur.com/9WoshUg.png" alt="alt text" width="300" height="auto" />
<img src="https://i.imgur.com/OSZiJET.png" alt="alt text" width="300" height="auto" />|<br/><img src="https://i.imgur.com/zk4XxJp.png" alt="alt text" width="300" height="auto" />
<img src="https://i.imgur.com/AlcwgKT.png" alt="alt text" width="300" height="auto" />|<img src="https://i.imgur.com/z0d1eZt.png" alt="alt text" width="300" height="auto" />
<br/>

## Testing Instructions






This PR can be tested by following these steps:

Important! From Pascal: FYI 3P PRs are not uploaded for QA due to GHA limitations. One would need to download the ZIP file from https://github.com/google/web-stories-wp/runs/4011337300 and install that version manually.

1. Add media (test with both image and video)
2. Open the Filter Panel, verify that in case of "Tint", eyedropper is displayed next to the Color Picker and it works as expected. There should be no eyedropper for Linear / Vignette.
3. Verify the style of the eyedropper looks as in [Figma ](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor---Sprint-Feature-Level-Designs_?node-id=10291%3A442662)
4. Open the Accessibility panel. Verify the description of the Accessibility field is displayed above the textarea for both Video and Image
5. Verify the design matches [Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor---Sprint-Feature-Level-Designs_?node-id=10250%3A436662)


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9465
